src/utils/initializeStripe/index.js

```diff
@@ -2,8 +2,5 @@ import { loadStripe } from '@stripe/stripe-js'
 
 export const initializeStripe = async () => {
```

Comment:
this can be shortened to 
```suggestion
export const initializeStripe = () => loadStripe(process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY)
```